pkg/cmd/cli/backuplocation/create.go

```diff
@@ -107,6 +109,10 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 		return errors.New("--backup-sync-period must be non-negative")
 	}
 
+	if o.StoreValidationPeriod < 0 {
```

Comment:
should we set this to default and continue?

Reply:
Not sure. What if the user for some reason expects that value to take, and we give them another value. Could this be possible?